src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  @include grid-template-columns(1);
+  gap: 48px;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (min-width: 488px) {
+    @include grid-template-columns(2);
+  }
+
+  @media (min-width: 768px) {
+    @include grid-template-columns(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include grid-template-columns(4);
+  }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.